draft-ietf-tls-dtls13.md

```diff
@@ -750,6 +765,55 @@ because an attacker will have difficulty forging a datagram that will
 not be rejected by the transport layer.
 
 
+### AEAD Limits
+
+Section 5.5 of TLS {{!TLS13}} defines limits on the number of records that can
+be protected using the same keys. These limits are specific to an AEAD
+algorithm, and apply equally to DTLS. Implementations SHOULD NOT protect more
+records than allowed by the limit specified for the negotiated AEAD.
+
+{{!TLS13}} does not specify a limit for AEAD_AES_128_CCM, but the analysis in
+{{ccm-bounds}} shows that a limit of 2^23 packets can be used to obtain the
+same confidentiality protection as the limits specified in TLS.
+
+The usage limits defined in TLS 1.3 exist for protection against attacks
+on confidentiality and apply to successful applications of AEAD protection. The
+integrity protections in authenticated encryption also depend on limiting the
+number of attempts to forge packets. TLS achieves this by closing connections
+after any record fails an authentication check. In comparison, DTLS ignores any
+packet that cannot be authenticated, allowing multiple forgery attempts.
+
+Implementations MUST count the number of received packets that fail
+authentication. If the number of packets that fail authentication exceeds a
+limit that is specific to the AEAD in use, an implementation MUST immediately
+close the connection. Implementations SHOULD initiate a key update before
+reaching this limit. Applying a limit reduces the probability that an attacker
+is able to successfully forge a packet; see {{AEBounds}} and {{ROBUST}}.
+
+For AEAD_AES_128_GCM, AEAD_AES_256_GCM, and AEAD_CHACHA20_POLY1305, the limit
+on the number of records that fail authentication is 2^36. Note that the
+analysis in {{AEBounds}} supports a higher limit for the AEAD_AES_128_GCM and
+AEAD_AES_256_GCM, but this specification recommends a lower limit. For
+AEAD_AES_128_CCM, the limit on the number of records that fail authentication
+is 2^23.5; see {{ccm-bounds}}.
+
+The AEAD_AES_128_CCM_8 AEAD, as used in TLS_AES_128_CCM_SHA256, does not have a
+limit on the number of records that fail authentication that both limits the
+probability of forgery by the same amount and does not expose implementations
+to the risk of denial of service; see {{ccm-short}}. Therefore,
+TLS_AES_128_CCM_SHA256 MUST NOT used in DTLS without additional safeguards
+against forgery. Implementations MUST set usage limits for AEAD_AES_128_CCM_8
+based on an understanding of any additional forgery protections that are used.
```

Comment:
Do we have some idea what those protections might be?

Reply:
My only thought here is that you might have a hard limit on the number of packets you can receive, ever that makes larger limits meaningless.

More realistically, this probably comes down to having a higher tolerance for forgeries in that application.  In the real-time case, the window over which a forgery has any effect is pretty narrow.  For instance, new video I-frames replace the entire scene.  So I can see how someone might justify a higher tolerance for forgery.